### PR TITLE
add gitignore

### LINKED ISSUE #1668: add .gitignore
### What would you like to be enhanced?

add .gitignore

### Why is this needed?

keep the repository clean by excluding temporary files.

if it pass, i can do this.

### PR DESCRIPTION
**Pull request by SunJiFengPlus**
**What type of PR is this?**

enhance

**What this PR does / why we need it?**

keep the repository clean by excluding temporary files.

**Which issue(s) this PR fixes？**

Fixes #1668

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/sermant-io/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [ ] GitHub Actions works fine in this PR.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ide
+.idea/
+*.iml
+.vscode/
+
+# debug
+logs
+arthas-output
```
Comment (lilai23) [inline .gitignore]: remove arthas-output

Comment (SunJiFengPlus) [reply .gitignore]: done

**Review by lilai23 — DISMISSED** [review]

**Review by daizhenyu — DISMISSED** [review]

**Review by provenceee — DISMISSED** [review]

**Comment by lilai23** [comment]
check line lint action

**Comment by SunJiFengPlus** [comment]
What are the rules for checking line lint? I checked that my line sperator is LF(\n), do I need to end with a blank line?

**Review by lilai23 — APPROVED** [review]

**Review by hanbingleixue — APPROVED** [review]

**Pull request merged**